### PR TITLE
Add bio for jonhoo

### PR DESCRIPTION
**Pull request by jonhoo**

### CODE REVIEW

File: mentors/rust/j.json [hunk 1]
```diff
@@ -15,5 +15,14 @@
     "link_url": "https://cslai.coolsilon.com/",
     "avatar_url": "https://www.gravatar.com/avatar/3c09250bd1e6c4618d8e469e7aa0468d",
     "bio": "Writes Javascript/ES and Python at work, discovered Rust and find it expressive despite being a system langauge. Would love to contribute back by mentoring."
+  },
+
```
Comment (rpottsoh) [inline mentors/rust/j.json]: Please remove blank line.

```suggestion
```

**Comment by jonhoo** [comment]
I added it there because there was already one between the existing two bios, but removed it now :)

**Comment by rpottsoh** [comment]
Ic that now.  Can you squash your two commits and then add another commit that removes the lingering blank line.  When Travis CI ever gets done I'll merge this.

**Comment by jonhoo** [comment]
@rpottsoh done!

**Comment by rpottsoh** [comment]
Perfect, thanks.....

**Pull request merged**